FP_NuTemplate_Cro.VR/config.sqf

```diff
@@ -0,0 +1,11 @@
+// Only allow people with map tools to draw on briefing (true = everyone can draw)
+ACE_maptools_EveryoneCanDrawOnBriefing = false;
```

Comment:
Didn't this variable and the ACE Linedrawing get removed in favor of the Vanilla functionality?